spec/factories/texting_recipients.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :texting_recipient do
+    recipient_name { "MyString" }
+    phone_number { "MyString" }
```

Comment:
It might be good to generate fake data that looks more like an actual phone number. We use a library named [`Faker`](https://github.com/faker-ruby/faker) which is useful for generating different kinds of structured data, and they even have a method for generating fake phone numbers. See https://github.com/ShelterTechSF/askdarcel-api/blob/de32f656c19116507d0ab9f35a58eadd88583f12/spec/factories/phones.rb#L11 for where we currently use that function for other testing factories.